package.json

```diff
@@ -22,6 +22,7 @@
     "generate-component-statuses": "./utilities/component-statuses/generate-component-statuses.js"
   },
   "scripts": {
+    "analyse-component-bundles": "turbo run analyse-component-bundles --filter=\"./packages/components/**\" --filter=!pie-webc --filter=!pie-webc-testing --filter=!pie-webc-core",
```

Comment:
No need for these filters, as turbo by default only executes tasks if present in a package.json